buildtest/cli/__init__.py

```diff
@@ -82,6 +83,35 @@ def positive_number(value):
     return int_val
 
 
+def supported_color(value):
+    """Checks if input is positive number and returns value as an int type.
+
+    Args:
+        value (str): Specify an input number
+
+    Returns:
+        str: Return value as str type
+
+    Raises:
+        argparse.ArgumentTypeError will be raised if input is not a supported color input or is not str type
+
+    >>> supported_color("red")
+    red
+    """
+
+    if not isinstance(value, (str)):
+        raise argparse.ArgumentTypeError(
+            f"Input must be a string type, you have specified '{value}' which is of type {type(value)}"
+        )
+    try:
+        color_val = Color.parse(value)
+    except ColorParseError:
+        console.print(f"[red]Unable to convert {value} to a Color ")
+        console.print_exception()
+        return None
```

Comment:
note that `return None` is same as just doing a `return`.

I think if the exception is raised this is the correct behavior from this method. I was contemplating on using a default color if there is an error but i think its probably better we raise an exception on invalid color type instead of resorting to a default color.

Reply:
simplified